### DISCUSSION TITLE
Can I cancel generating image by api

### DISCUSSION THREAD

**Discussion by Xiuzhenpeng** [mcmonkeyprojects/SwarmUI#409] [Q&A]
I like swarmui very much. And i want to make a webapp, based on swarmui.
How can i cancel generating image by api. I have read api.md many times, i still donnot find this api.
Or there's no api for that ?

**Comment by mcmonkey4eva** [comment]
Everything you can do in the UI, you can do in the API. Press the interrupt button in the UI and look in browser tools at what network call it made.

**Reply by Xiuzhenpeng** [reply]
thank you very much, i have found it yesterday. So i closed this discussion page.
